lib/cli/get-files.js

```diff
@@ -0,0 +1,50 @@
+const { readdirSync, statSync } = require('fs');
+const { join } = require('path');
+
+/**
+ * Returns a list of usable extensions from a comma separated list
+ * @param {string} extensions
+ * @returns {[]string}
```

Comment:
```suggestion
 * @returns {string[]}
```
I'm not sure if `[]string` will work, even if - for consistency - better to use `string[]`

Reply:
Hey, thanks for this. I think you have picked an old commit, as this should have been already solved.